csi/identity.go

```diff
@@ -22,8 +22,8 @@ import (
 )
 
 const (
-	csiDriverVersion = "0.1.0"
-	csiDriverName    = "csi-osd"
+	csiDriverVersion    = "0.1.0"
+	csiDriverNamePrefix = "com.portworx.openstorage."
```

Comment:
The prefix does not need to have portworx in it? CSI/osd should be driver agnostic.
It could simply be "com.openstorage."

Reply:
Great point. I will change that.